src/components/CurriculumComponents/CurriculumDownloadTab/index.tsx

```diff
@@ -321,7 +323,20 @@ const CurriculumDownloadTab: FC<CurriculumDownloadTabProps> = ({
 
   return (
     <OakThemeProvider theme={oakDefaultTheme}>
-      <Box $maxWidth={1280} $mh={"auto"} $ph={18} $pb={[48]} $width={"100%"}>
+      <Box
+        id="curriculum-downloads"
+        aria-labelledby="curriculum-downloads-heading"
```

Comment:
See previous comment re: use of `aria-labelledby`

Reply:
As above,  I've added the role attribute